ci/pipelines/deploy-to-production.yml

```diff
@@ -12,6 +12,7 @@ definitions:
   deploy_params: &deploy_params
     <<: *aws_assumed_role_creds
     TAG: ((.:tag))
+    NGINX_IMAGE_TAG: ((.:nginx_image_tag))
```

Comment:
I do wonder if we should rename `TAG` to `APPLICATION_IMAGE_TAG` (not in this PR though!)

Reply:
We definitely should! I didn't do it before because would break some stuff but I'll be able to do it after this story is finished.

Reply:
Do we want to include `NGINX_IMAGE_TAG` in the `wait_for_deploy_params` as well?

Reply:
Good spot! Yes - all the wait-for-deploy.js script is doing is just printing out the value of `TAG` so probably worth having the version of nginx alongside with it as well. Frontend is unique as it has an additional nginx proxy so might have to modify wait-for-deploy.js to display the 2nd nginx as well...